tracing/tracing.go

```diff
@@ -1,59 +1,81 @@
 // Package tracing handles opentracing support for skipper
+//
+// The tracers, except for "noop", are build as loadable modules. The modules must have
```

Comment:
Let's consistently call these Plugins. Here, since it's the first time, I'd:
s/loadable modules/Go Plugins
s/The modules/The plugins

Reply:
We should also mention the note from Go's plugin package that this is currently incomplete and only supported on Linux